src/prefect/utilities/serialization.py

```diff
@@ -231,6 +231,23 @@ def _serialize(self, value, attr, obj, **kwargs):  # type: ignore
         return super()._serialize(value, attr, obj, **kwargs)
 
 
+class SortedList(fields.List):
+    """
+    An extension of the Marshmallow List field that sorts the serialized object for
+    determinism
+
+    Args:
+        - cls_or_instance (type): the inner field class
+        - **kwargs (Any): the keyword arguments accepted by `marshmallow.Field`
+    """
+
+    def __init__(self, cls_or_instance: Union[fields.Field, type], **kwargs: Any):
+        super().__init__(cls_or_instance, **kwargs)
```

Comment:
If a class doesn't override the `__init__` at all, there's no need to define one.

```suggestion
```